docs/usage/modules/manager/index.md

```diff
@@ -18,38 +18,38 @@ The goal of Renovate is to detect and maintain all third-party dependencies in y
 
 ### File Matching
 
-Most managers have a default `fileMatch` array.
-The `fileMatch` array has regular expression strings that match against the repository file list.
+Most managers have a default `filePatterns` array.
+The `filePatterns` array has regular expression or glob pattern that matches against the repository file list.
 
-#### Managers with no default fileMatch
+#### Managers with no default filePatterns
 
-Some managers have no default `fileMatch` regular expression, because they have no filename convention that would let Renovate intelligently filter them.
-In such a case, the manager will be disabled until you create a `fileMatch` regular expression, e.g. like the following:
+Some managers have no default `filePatterns`, because they have no filename convention that would let Renovate intelligently filter them.
+In such a case, the manager will be disabled until you create a `filePatterns` regular expression, e.g. like the following:
```

Comment:
```suggestion
If there is no default `filePattern`, the manager is disabled.
For the manager to work, you must create a `filePatterns` regular expression, or glob pattern.
For example:
```

Please double check that the `filePattern` accepts a glob pattern, before applying this suggestion. 😉